### PR TITLE
afl-fuzz 2.30b

### PR DESCRIPTION
**Pull request by alex**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/afl-fuzz.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@ def install
     # behaving correctly!" in a nested login shell.
     # Reported to lcamtuf@coredump.cx 6th Apr 2016.
```
Comment (MikeMcQuaid) [inline Formula/afl-fuzz.rb]: Could you try to submit a patch to them? It doesn't look like they are working on fixing this.

Comment (alex) [inline Formula/afl-fuzz.rb]: Sent an email to the mailing list asking about it, I'm not up for drafting a patch myself, as I don't understand the root cause. (Unless you just mean a patch do the same thing as this find/replace)

Comment (MikeMcQuaid) [inline Formula/afl-fuzz.rb]: Let's just remove the patch for now, then, thanks!

Comment (alex) [inline Formula/afl-fuzz.rb]: If this `inreplace` is removed then `afl-fuzz` does not build, as far as I understand.

Comment (MikeMcQuaid) [inline Formula/afl-fuzz.rb]: Can you try it and see if it builds for you? If it does, we'll see if it builds on CI and if it does: we can remove the patch 😀

Comment (alex) [inline Formula/afl-fuzz.rb]: Confirmed that it built locally, trying on CI now.

**Comment by alex** [comment]
@MikeMcQuaid Ok, looks to be clean after all! I guess it was fixed and no one noticed.

**Comment by MikeMcQuaid** [comment]
Thanks again @alex! This is why I hate patching 😀

**Pull request closed without merging**